src/main/java/org/opendatasoft/elasticsearch/ingest/GeoExtensionProcessor.java

```diff
@@ -265,10 +267,33 @@ public IngestDocument execute(IngestDocument ingestDocument) throws IOException,
             if (centroidField != null) ingestDocument.setFieldValue(
                     geoShapeField + "." + centroidField, GeoUtils.getCentroidFromGeom(geom));
             if (bboxField != null) {
-                Coordinate[] coords = geom.getEnvelope().getCoordinates();
-                if (coords.length >= 4) ingestDocument.setFieldValue(
-                        geoShapeField + "." + bboxField,
-                        GeoUtils.getBboxFromCoords(coords));
+                Coordinate[] coords;
+                if (geom.getGeometryType() == "Point") {
```

Comment:
String comparison is usually a bit costly. Isn't there another way to test for the point type ?

Reply:
https://www.scaler.com/topics/difference-between-equals-method-in-java/

```suggestion
                if (Objects.equals(geom.getGeometryType(), "Point")) {
```